ros2_ws/src/warehouse_robot/urdf/robot.urdf.xacro

```diff
@@ -11,36 +11,30 @@
 
   <xacro:include filename="$(find warehouse_robot)/urdf/robot_parts/traction_wheel.xacro" />
 
-  <xacro:include filename="$(find warehouse_robot)/urdf/robot_parts/tray.xacro" />
+  <!-- <xacro:include filename="$(find warehouse_robot)/urdf/robot_parts/tray.xacro" /> -->
 
-  <xacro:include filename="$(find warehouse_robot)/urdf/robot_parts/piston.xacro" />
+  <!-- <xacro:include filename="$(find warehouse_robot)/urdf/robot_parts/piston.xacro" /> -->
 
   <xacro:include filename="$(find warehouse_robot)/urdf/robot_parts/omni_wheel.xacro" />
 
   <!--=========================== SENSORS ===========================-->
 
-  <xacro:arg name="sensor_imu" default="true" />
-
-  <xacro:arg name="sensor_lidar" default="true" />
+  <!-- <xacro:arg name="sensor_imu" default="true" /> -->
 
   <xacro:arg name="sensor_rplidar" default="true" />
 
-  <xacro:arg name="sensor_camera" default="true" />
+  <!-- <xacro:arg name="sensor_camera" default="true" /> -->
 
-  <xacro:if value="$(arg sensor_imu)">
+  <!-- <xacro:if value="$(arg sensor_imu)">
     <xacro:include filename="$(find warehouse_robot)/urdf/sensors/imu.xacro" />
-  </xacro:if>
-
-  <xacro:if value="$(arg sensor_lidar)">
-    <xacro:include filename="$(find warehouse_robot)/urdf/sensors/lidar.xacro" />
-  </xacro:if>
+  </xacro:if> -->
 
   <xacro:if value="$(arg sensor_rplidar)">
     <xacro:include filename="$(find warehouse_robot)/urdf/sensors/rplidar.xacro" />
   </xacro:if>
 
-  <xacro:if value="$(arg sensor_camera)">
+  <!-- <xacro:if value="$(arg sensor_camera)">
```

Comment:
should we just delete this instead of commenting it out?

Reply:
Actually, I should uncomment this. I just commented it out during debugging.